.editorconfig

```diff
@@ -0,0 +1,16 @@
+# editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = tab
```

Comment:
Big fan of editorconfig, but can we switch to 2 spaces, rather than tabs? HTML gets very spread out with tabs.